protocol/x/clob/keeper/orders.go

```diff
@@ -496,6 +496,7 @@ func (k Keeper) PlaceStatefulOrdersFromLastBlock(
 	ctx sdk.Context,
 	placedStatefulOrderIds []types.OrderId,
 	existingOffchainUpdates *types.OffchainUpdates,
+	onlyPlacePostOnly bool,
```

Comment:
Nit: current naming might suggest that if `false`, the function processes ALL orders including post-only. How abt sth like `postOnlyFilter` and comment explicitly what this var does?